### DISCUSSION TITLE
Failed to load static props when making a new project

### DISCUSSION THREAD

**Discussion by Nit-Twit** [vercel/community#7255] [Bug Report]
### Page to Investigate

https://vercel.com/new

### Steps to Reproduce

1) Make a new project
2) Wait for it to load
![image](https://github.com/vercel/community/assets/157034077/12e12a90-c61f-4332-aa89-53501dfe6848)

**Comment by leerob** [comment]
Hey there, are you still seeing this issue? I cannot reproduce it consistently, but we will look into the error logs to see if there's a bug we can patch here.

**Reply by Nit-Twit** [reply]
Yes, I am still seeing this error

**Reply by SoCoVentures** [reply]
and I am seeing the same issue on my project as well

**Reply by karangoraniya** [reply]
I am also getting same error while deploying new project
<img width="1504" alt="Screenshot 2024-06-22 at 2 37 56 AM" src="https://github.com/vercel/community/assets/61357747/5e924e4b-9ef1-4483-a1a4-60d5d386b4b3">

**Reply by Will-20** [reply]
Also seeing the same error when starting a new repo

**Reply by moods-1** [reply]
ditto

**Reply by SamVerse** [reply]
> Hey there, are you still seeing this issue? I cannot reproduce it consistently, but we will look into the error logs to see if there's a bug we can patch here.

Also facing similar issue

**Reply by HollosJ** [reply]
Same issue here! Console throwing 500s

**Comment by sikka-sa** [comment]
whew I thought it was only me

**Comment by leerob** [comment]
Thank you all, we can now confirm this issue and are actively looking into this 🙏

**Reply by moods-1** [reply]
All good here. Thanks!

**Comment by sikka-sa** [comment]
It seems to be fixed now

**Reply by Will-20** [reply]
Yep fixed for me as well

**Reply by HollosJ** [reply]
Yup. All good! Thanks @leerob 🙏

**Comment by federicoruf** [comment]
Is the Vercel team still looking for a solution? Issue is happening to me

**Comment by Megaemce** [comment]
Bug seems to be alive again! 🐛 :goberserk:

**Reply by poinch** [reply]
yep same for me

**Comment by hasdfa** [comment]
Still have the same issue.

```
Error: Failed to load static props
    at https://vercel.com/_next/static/chunks/main-b36d222eec12eb1d.js?dpl=dpl_Hv2DsTqf5NjRPWk7Xf2KVjfstAQV:3:87080
    at async F (https://vercel.com/_next/static/chunks/main-b36d222eec12eb1d.js?dpl=dpl_Hv2DsTqf5NjRPWk7Xf2KVjfstAQV:3:83702)
    at async z.getRouteInfo (https://vercel.com/_next/static/chunks/main-b36d222eec12eb1d.js?dpl=dpl_Hv2DsTqf5NjRPWk7Xf2KVjfstAQV:3:97324)
    at async z.change (https://vercel.com/_next/static/chunks/main-b36d222eec12eb1d.js?dpl=dpl_Hv2DsTqf5NjRPWk7Xf2KVjfstAQV:3:92551)
```

![Screenshot 2024-07-08 at 14 06 48](https://github.com/vercel/community/assets/12295460/42341fd3-5ba3-417c-9094-8fe500741ae5)

**Comment by fsevindik** [comment]
pfffffffffffffff
![Screenshot 2024-07-08 160946](https://github.com/vercel/community/assets/61114943/f6127511-14fd-45cd-bad6-756928fbfe98)
![Screenshot 2024-07-08 160914](https://github.com/vercel/community/assets/61114943/424e0058-91d5-45ba-abe4-f436070aa067)

[a comment by an automated account is not shown]
